config/test.exs

```diff
@@ -5,81 +5,84 @@ config :logger, :console,
   format: "\n$time $metadata[$level] $levelpad$message\n",
   metadata: [:query_time, :response_status]
 
+common_host = [
+  database:  (System.get_env("INSTREAM_DATABASE") || "instream_test"),
```

Comment:
Setting the database as a default one should also clean up the tests individually passing one, i.e. `Instream.ConnectionTest`. Otherwise it would sometimes not work as expected.